### ISSUE TITLE
[BUG] Cant setup storage account - Hostname/IP does not match certificate's altnames

### ISSUE THREAD

**Issue by PepeGunSSC** [GenesysGo/shdw-drive-bug-reports#1]
**Describe the bug**
bigint: Failed to load bindings, pure JS will be used (try npm run rebuild?)
This is beta software. Use at your own discretion.
✔ By creating your first storage account on Shadow Drive, you agree to the Terms of Service as outlined here: https://shadowdrive.org. Confirm? … yes
✔ This storage account will require an estimated 0.01048576 SHDW to setup. Would you like to continue? … yes
✖ Error processing transaction. See below for details:
FetchError: request to https://solana.shadow.genesysgo.net/storage-account failed, reason: Hostname/IP does not match certificate's altnames: Host: solana.shadow.genesysgo.net. is not in the cert's altnames: DNS:*.genesysgo.net, DNS:genesysgo.net
    at ClientRequest.<anonymous> (/snapshot/shdw-drive-cli/node_modules/node-fetch/lib/index.js:1491:11)
    at ClientRequest.emit (node:events:526:28)
    at TLSSocket.socketErrorListener (node:_http_client:442:9)
    at TLSSocket.emit (node:events:526:28)
    at emitErrorNT (node:internal/streams/destroy:157:8)
    at emitErrorCloseNT (node:internal/streams/destroy:122:3)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  type: 'system',
  errno: 'ERR_TLS_CERT_ALTNAME_INVALID',
  code: 'ERR_TLS_CERT_ALTNAME_INVALID'
}

**To Reproduce**
shdw-drive create-storage-account --keypair ~/.config/solana/id.json --name "My Storage Account" --size 10MB

**Expected behavior**
A storage account opened

**Screenshots**
![image](https://user-images.githubusercontent.com/105282103/167604809-7732889e-e641-472b-b560-f486c0fa8376.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04.4 LTS

**Additional context**
Trying to upload the first immutable pepegun

**Comment by DataKnox** [comment]
Can confirm I am also experiencing this issue
`✖ Error processing transaction. See below for details:
FetchError: request to https://solana.shadow.genesysgo.net/storage-account failed, reason: Hostname/IP does not match certificate's altnames: Host: solana.shadow.genesysgo.net. is not in the cert's altnames: DNS:*.genesysgo.net, DNS:genesysgo.net
    at ClientRequest.<anonymous> (/snapshot/shdw-drive-cli/node_modules/node-fetch/lib/index.js:1491:11)
    at ClientRequest.emit (node:events:526:28)
    at TLSSocket.socketErrorListener (node:_http_client:442:9)
    at TLSSocket.emit (node:events:526:28)
    at emitErrorNT (node:internal/streams/destroy:157:8)
    at emitErrorCloseNT (node:internal/streams/destroy:122:3)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  type: 'system',
  errno: 'ERR_TLS_CERT_ALTNAME_INVALID',
  code: 'ERR_TLS_CERT_ALTNAME_INVALID'
}`

**Comment by na** [comment]
Same here

```
✖ Error processing transaction. See below for details:
FetchError: request to https://solana.shadow.genesysgo.net/storage-account failed, reason: Hostname/IP does not match certificate's altnames: Host: solana.shadow.genesysgo.net. is not in the cert's altnames: DNS:*.genesysgo.net, DNS:genesysgo.net
    at ClientRequest.<anonymous> (/snapshot/shdw-drive-cli/node_modules/node-fetch/lib/index.js:1491:11)
    at ClientRequest.emit (node:events:526:28)
    at TLSSocket.socketErrorListener (node:_http_client:442:9)
    at TLSSocket.emit (node:events:526:28)
    at emitErrorNT (node:internal/streams/destroy:157:8)
    at emitErrorCloseNT (node:internal/streams/destroy:122:3)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  type: 'system',
  errno: 'ERR_TLS_CERT_ALTNAME_INVALID',
  code: 'ERR_TLS_CERT_ALTNAME_INVALID'
}
```

**Comment by tracy-codes** [comment]
@na @PepeGunSSC Can you try downloading the latest binary from https://shdw.genesysgo.com/using-shadow-drive/the-shadow-drive-platform/shadow-drive-cli#installation and let me know if this persists?

**Comment by CoachChuckFF** [comment]
> @na @PepeGunSSC Can you try downloading the latest binary from https://shdw.genesysgo.com/using-shadow-drive/the-shadow-drive-platform/shadow-drive-cli#installation and let me know if this persists?

That seemed to solve the issue! (on the MacOS x64 build ) For others trying to fix this, visit the [link](https://shdw.genesysgo.com/using-shadow-drive/the-shadow-drive-platform/shadow-drive-cli#installation) incognito to make sure you have the most up-to-date binary.

**Comment by na** [comment]
Confirmed issue solved on Windows 11 with WSLv1 - Ubuntu-20.04

**Comment by robbestad** [comment]
~Same problem on linux (using 0.0.1)~

Hard-refreshing the url as suggested above gave me a new binary that worked.

**Comment by tracy-codes** [comment]
Looks like this has now been resolved via the latest version (0.0.3) that's available on the GitBook. Thank you for reporting.
